invenio_oaiserver/response.py

```diff
@@ -257,7 +254,10 @@ def listidentifiers(**kwargs):
 
 def listrecords(**kwargs):
     """Create OAI-PMH response for verb ListIdentifiers."""
-    from invenio_records.models import RecordMetadata
+    kwargs['verb'] = 'ListRecords'
```

Comment:
Actually the [verb()](https://github.com/jirikuncar/invenio-oaiserver/blob/restart/invenio_oaiserver/response.py#L99) function require it and I don't think make sense pass another kind of verb (or there is some special case?).